driver-core/src/test/functional/com/mongodb/client/test/CollectionHelper.java

```diff
@@ -368,8 +368,12 @@ public void createIndex(final Bson key, final Double bucketSize) {
     }
 
     public void killAllSessions() {
-        new CommandWriteOperation<BsonDocument>("admin", new BsonDocument("killAllSessions", new BsonArray()),
-                new BsonDocumentCodec());
+        try {
+            new CommandWriteOperation<BsonDocument>("admin", new BsonDocument("killAllSessions", new BsonArray()),
+                    new BsonDocumentCodec()).execute(getBinding());
```

Comment:
This was never executing... Sometimes it helps to actually do it in order to clean up after a failing test

Reply:
:)